### ISSUE TITLE
Autocorrect?

### ISSUE THREAD

**Issue by duomoteng** [memspace/zefyr#283]
Does zefyr support for autocorrect now? Or is there any suggestion to achieve it by myself?

**Comment by cgestes** [comment]
you mean implementing this callback ```showAutocorrectionPromptRect```?

**Comment by duomoteng** [comment]
I guess yes! For example, Highlight the text that require auto correct.

**Comment by cgestes** [comment]
We should implement showAutocorrectionPromptRect, for that, it is available only on Mac/iOS for now.

**Comment by duomoteng** [comment]
Yes! It seems that it will solve my issue. Can you give me some hint hot to implement it?

**Comment by duomoteng** [comment]
Besides, I think in android the auto correction still exist, does it require another way to achieve it?

**Comment by cgestes** [comment]
the showAutocorrectionPromptRect is for showing a visual indication.

I dont know about the autocorrection state, doesn't it work already?

**Comment by duomoteng** [comment]
Sorry, I mean that it seems that showAutocorrectionpromptrect already exists in Android.
<img width="562" alt="Screen Shot 2020-05-06 at 21 05 39" src="https://user-images.githubusercontent.com/48649432/81180256-57b14080-8fdd-11ea-8b16-1663a3785cbd.png">
